action_plugins/play_sound/__init__.py

```diff
@@ -141,28 +141,36 @@ def __init__(
         super().__init__(behavior_type)
 
         # Model variables
-        self.sound_filename: str = ""
-        self.sound_volume: int = 50
+        self.soundFilename: str = ""
```

Comment:
For consistency with the rest of Gremlin it would be preferrable for these to be `sound_filename` and `sound_volume`

Reply:
n/p, I think I got better understanding now how QML action properties and data class work.